base/ca/src/main/java/com/netscape/cms/servlet/cert/scep/CRSEnrollment.java

```diff
@@ -268,7 +268,7 @@ public void init(ServletConfig sc) {
             mHashAlgorithm = scepConfig.getString("hashAlgorithm", "SHA1");
```

Comment:
@ckelleyRH I forgot to mention this.  On #cs, I saw the question and replied.   This could be safely changed to SHA256 since it's configurable. Thanks.

Reply:
OK, thanks! I've made that change and will merge.